dev/com.ibm.ws.springboot.fat30.actuator.app/src/main/java/com/ibm/ws/springboot/fat30/actuator/app/TestApplication.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2018,2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.springboot.fat30.actuator.app;
```

Comment:
```suggestion
package io.openliberty.springboot.fat30.actuator.app;
```
Since it is a new project, it should have io.openliberty package.